### PR TITLE
Andy lesson merge from personal fork test

### PR DESCRIPTION
**Pull request by loluwot**

### CODE REVIEW

**Comment by loluwot** [comment]
Organic chemistry lesson.

File: content/chemistry/index.md [hunk 1]
```diff
@@ -4,5 +4,5 @@ title: Chemistry
 nav_order: 0
 nav_exclude: true
 ---
-Welcome to our chemistry guide.
+Welcome to our chemistry guides.
```
Comment (jaegpark) [inline content/chemistry/index.md]: revert this back, this was a test

File: content/chemistry/organic-chemistry/Nucleophiles,-Electrophiles-and-Arrow-Pushing.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
+layout: lesson
+author: Andy Cai
+title: Nucleophiles, Electrophiles and Arrow Pushing Notation
+parent: Organic Chemistry
+---
```
Comment (jaegpark) [inline content/chemistry/organic-chemistry/Nucleophiles,-Electrophiles-and-Arrow-Pushing.md]: add
`grand_parent: Chemistry`

**Review by jaegpark — CHANGES_REQUESTED** [review]

**Review by jaegpark — APPROVED** [review]

**Comment by jaegpark** [comment]
andy lesson update 2

**Pull request merged**